app/users/userprofile/userprofile.component.ts

```diff
@@ -52,6 +81,10 @@ export class ManageUserComponent implements OnInit {
       );
   }
 
+  passwordValid() {
+    return this.user.confirmPass.match(/^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{6,}$/);
```

Comment:
*What the fuck?! Why are we doing this?*

Reply:
I did not do it..it was there already...